### ISSUE TITLE
Problems about compiling spaint

### ISSUE THREAD

**Issue by MarkZurkberg** [torrvision/spaint#1]
I am trying to compile spaint on my computer and I wonder if it is necessary that I should compile InfiniTAM  successfully first. Thanks! @torrvision

**Comment by sgolodetz** [comment]
Yes, InfiniTAM is a compulsory dependency - you can't compile SemanticPaint without it.

**Comment by MarkZurkberg** [comment]
Thanks to your help, and I have run InfiniTAM-spaint_v2 successfully on my computer. These days I am trying to compile spaint and got errors in configuring process. The error are as follows:
Could NOT find PkgConfig (missing:  PKG_CONFIG_EXECUTABLE) 
CMake Error at C:/Program Files (x86)/CMake/share/cmake-3.4/Modules/FindBLAS.cmake:681 (message):
  A required library with BLAS API not found.  Please specify library
  location.
Call Stack (most recent call first):
  cmake/FindArrayFire.cmake:77 (FIND_PACKAGE)
  cmake/UseArrayFire.cmake:9 (FIND_PACKAGE)
  modules/spaint/CMakeLists.txt:21 (INCLUDE)
@sgolodetz 
![spaint](https://cloud.githubusercontent.com/assets/8450271/14675193/62b55d2a-073a-11e6-8931-3fe7d4de6777.png)

**Comment by sgolodetz** [comment]
Look under the Advanced options - if I remember right, you need to specify one of the libraries on which ArrayFire depends. (Also, I do get notified when you submit issues on GitHub - no need to email everybody in my research group including my supervisor. Incidentally, when I say "no need", I mean "please don't", for clarity :))

**Comment by maoLB** [comment]
hello,MarkZurkberg. I have the same problem with you. How do you solve this problem?
